test/e2e/storage/csimock/csi_volume_expansion.go

```diff
@@ -396,7 +394,7 @@ var _ = utils.SIGDescribe("CSI Mock volume expansion", func() {
 		}
 	})
 
-	f.Context("Expansion with recovery", feature.RecoverVolumeExpansionFailure, framework.WithFeatureGate(features.RecoverVolumeExpansionFailure), func() {
```

Comment:
The feature label `RecoverVolumeExpansionFailure` can be removed from https://github.com/kubernetes/kubernetes/blob/master/test/e2e/feature/feature.go#L367

Reply:
This is fixed now.